pages/Guides/supernode-program/docs.en.md

```diff
@@ -8,90 +8,89 @@ taxonomy:
 
 ## About Supernodes
 
-The NEM Supernode's program is funded with XEM set aside during the Nemesis block. These awards are then given to high performance nodes for helping to secure the network. These nodes form a backbone of support for [light wallets](http://nem.ghost.io/lightwallet/), mobile wallets, and 3rd party apps so that users of these services might have access to the network that is easy, fast, and reliable without having to sync a blockchain by themselves or use untrustworthy centralized services.
+The NEM Supernode's Program is funded with XEM set aside during the Nemesis block. These rewards are then given to high performance nodes for helping to secure the network. These nodes form a backbone of support for light wallets, mobile wallets, and 3rd party apps so that users of these services might have access to the network that is easy, fast, and reliable without having to sync a blockchain by themselves or use untrustworthy centralized services.
 
 The NEM network has been designed from the very beginning with the goal in mind that any light client can securely connect to and use any server safely to make any transaction. Supernodes are expected to be high performance and reliable nodes. They are regularly tested on their bandwidth, chain height, chain parts, computing power, version, ping, and responsiveness to make sure that they are performing to high standards. If they meet all these requirements, they are randomly given rewards.
 
-The Supernodes program is currently undergoing alpha testing and is looking for volunteer nodes to participate. During this phase of testing, rewards will **not** be distributed. Once the Supernode network is working and all parameters configured properly a date for the official start will be announced.
-
 ## Steps to Enroll and Participate in Supernodes
 
-- **Step 1**: Download and run the standalone version of NEM from [the GitHub releases page](https://github.com/NemProject/NanoWallet/releases).
+- **Step 1**: Download and run NEM Wallet from [the GitHub releases page](https://github.com/NemProject/NanoWallet/releases).
 
-  The standalone tutorials for Mac, Ubuntu, and Windows can be found at the [NEM Tutorials Index](http://nem.ghost.io/nem-tutorial-list/).
+  You can find tutorials on how to use NEM Wallet [here](https://nemproject.github.io/nem-docs/pages/Guides/nanowallet/docs.en.html).
 
-- **Step 2**: Make an account and have at least 3,000,030 XEM in an account.
+- **Step 2**: Make an account and have at least 10,010 XEM.
```

Comment:
Make sure to use [Title Case](https://en.wikipedia.org/wiki/Title_case) for each Step.